oqsprov/oqs_kmgmt.c

```diff
@@ -314,9 +314,25 @@ static int oqsx_export(void *keydata, int selection, OSSL_CALLBACK *param_cb,
     return ok;
 }
 
-#define OQS_KEY_TYPES()                                        \
-    OSSL_PARAM_octet_string(OSSL_PKEY_PARAM_PUB_KEY, NULL, 0), \
-        OSSL_PARAM_octet_string(OSSL_PKEY_PARAM_PRIV_KEY, NULL, 0)
+#define OSSL_HYBRID_PKEY_PARAM_CLASSICAL_PUB_KEY \
```

Comment:
Prefixing new params with "OSSL_" conceptually doesn't look OK as those should be managed by the OpenSSL team --> please change over to OQS_...